runtime/bindings/python/tests/test_inference_engine/test_core.py

```diff
@@ -318,6 +318,84 @@ def test_add_extension(device):
     # assert np.allclose(expected, computed[output_blob], atol=1e-2, rtol=1e-2)
 
 
+def test_py_extension(device):
```

Comment:
This is a wrong file for tests. You should add these to compatibility folder as you stated that Cython API is used: _/openvino/src/bindings/python/**tests_compatibility**/test_inference_engine/test_IECore.py_

CC @akuporos